### PR TITLE
Document Cleanup and Restructure

### PR DESCRIPTION
**Pull request by g00b**
- Separated Docs between LunaDefend and LunaTrace products, added README Documentanion for LunaTrace and link in docs.
- Added Legacy Code Context for Luna Defend
- Added new Sidebar and Category for LunaTrace

### CODE REVIEW

File: docs/blog/2021-10-18-announcing-lunasec.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ offering enterprise and open-source support for LunaDefend.
 
 AWS, NodeJS, and React is the fully supported stack at the moment, and now we are hard at work adding support for more environments.
 
-Please take a moment to learn about LunaDefend by reading the rest of this post and trying the [example app](https://www.lunasec.io/docs/pages/overview/demo-app/overview/) (it's really easy).
+Please take a moment to learn about LunaDefend by reading the rest of this post and trying the [example app](https://www.lunasec.io/docs](/pages/lunadefend/overview/demo-app/overview/) (it's really easy).
```
Comment (freeqaz) [inline docs/blog/2021-10-18-announcing-lunasec.md:31]: It looks like you've got a typo everywhere you replaced this. You don't need the `](` between `docs` and `/pages`. You just need it to be `https://www.lunasec.io/docs/pages/lunadefend/overview/demo-app/overview/` :ok_hand:

File: docs/pages/lunadefend/getting-started/introduction.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ sidebar_position: 1
 -->
 
 :::tip Try out the live demo
-The easiest way to see LunaSec in action is to check out our [Live Demo app](https://app.lunasec.dev).
+The easiest way to see LunaDefend in action is to check out our [Live Demo app](https://app.lunasec.dev).
 Just create a new user, input some data,
-and try out LunaSec.  To understand what is happening under the hood and what makes this special, read our [demo app documentation](https://www.lunasec.io/docs/pages/overview/demo-app/overview/).
+and try out LunaDefend.  To understand what is happening under the hood and what makes this special, read our [demo app documentation](https://www.lunasec.io/overview/demo-app/overview/).
```
Comment (freeqaz) [inline docs/pages/lunadefend/getting-started/introduction.md:24]: I think this link is busted

File: docs/blog/2021-12-12-log4j-zero-day-mitigation-guide.mdx [hunk 3]
```diff
@@ -402,7 +402,7 @@ _Spoiler: Maintained by us._
 
 An Open Source development framework that enables you to easily add a "Secure by Default" architecture to your existing
 software with only a few lines of code. It's loosely based on the OWASP Application Security Verification Standard 
-(above), and is also designed to be [adopted gradually](https://www.lunasec.io/docs/pages/how-it-works/security/introduction/).
+(above), and is also designed to be [adopted gradually](https://www.lunasec.io/docs](/pages/lunadefend/-it-works/security/introduction/).
```
Comment (g00b) [inline docs/blog/2021-12-12-log4j-zero-day-mitigation-guide.mdx:405]: Same fix, need to apply

File: docs/redirect-config.js [hunk 4]
```diff
@@ -40,50 +40,98 @@ const redirectConfig = {
       from: '/pages/getting-started/dedicated-tokenizer/backend/',
       to: '/pages/getting-started/dedicated-tokenizer/backend-setup/',
     },
+    {
```
Comment (g00b) [inline docs/redirect-config.js:43]: need, to adjust previous links, to new redirect

File: docs/blog/2021-10-18-announcing-lunasec.md (hunk 1 above)
Comment (g00b) [reply docs/blog/2021-10-18-announcing-lunasec.md]: Link should be fixed now

File: docs/pages/lunadefend/getting-started/introduction.md (hunk 2 above)
Comment (g00b) [reply docs/pages/lunadefend/getting-started/introduction.md]: Link should be fixed now

File: docs/pages/lunadefend/cli/start.md [hunk 5]
```diff
@@ -18,19 +18,21 @@ sidebar_position: 2
   ~
 -->
 
-The `lunasec start --env <environment>` command brings up the LunaSec stack in the provided environment configuration.
```
Comment (factoidforrest) [inline docs/pages/lunadefend/cli/start.md:21]: Sorry if I missed it and it's in here but let's add a one liner right when the CLI is first introduced..probably in cli/installation.md that says "This cli is called `lunasec` for legacy reasons.  It was created before we renamed the tool to LunaDefend"  

I know we were talking about a dedi file for that but it seems much simpler to do a quick one liner right on the CLI to me.  Much more likely to be read.

File: docs/pages/lunadefend/legacy/legacy-code.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  ~ https://creativecommons.org/licenses/by-sa/4.0/legalcode
+  ~
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+
+## Naming Convention Changes
+
+The original LunaDefend project started out under the project name LunaSec, as the company has grown and the product matured, we decided to formalize the product stack as LunaDefend. Because of this change you may find sections of the code and commands still referencing the previous lunasec. We wanted to give context to any users or developers curious as to our naming references and conventions.
```
Comment (factoidforrest) [inline docs/pages/lunadefend/legacy/legacy-code.md:23]: Ah here we go. As I mentioned above, this isn't enough content for its own page, and its out of the way and the people that need to read it, aren't going to.  Theyre just going to be confused and muddle through.  Let's try to pinpoint that place people will be confused while reading our docs and put this there.

Comment (factoidforrest) [reply docs/pages/lunadefend/legacy/legacy-code.md:23]: Excellently worded though

File: docs/pages/lunatrace/overview/introduction.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  ~ https://creativecommons.org/licenses/by-sa/4.0/legalcode
+  ~
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+## What is LunaTrace
+
+LunaTrace is a vulnerability scanner that tags and tracks inventories through to production deployment.  It combines static
+analysis at build time with runtime tracking and the capability to automatically patch some high-profile vulnerabilities.
```
Comment (factoidforrest) [inline docs/pages/lunatrace/overview/introduction.md:24]: I'm sure we will flesh this out a little more in the future. I have some ideas about what to put here so maybe ill make a pass on it later

**Review by factoidforrest — APPROVED** [review]
great PR and thanks for doing this. It must have been a little tediuos but I'm sure you warmed up to those refactoring tools pretty quick :)

File: docs/pages/lunadefend/legacy/legacy-code.md (hunk 6 above)
Comment (g00b) [reply docs/pages/lunadefend/legacy/legacy-code.md:23]: Maybe we just have a link to it from the pages where we start luna-sec

File: docs/pages/lunatrace/overview/introduction.md (hunk 7 above)
Comment (g00b) [reply docs/pages/lunatrace/overview/introduction.md:24]: Makes sense, I basically just am giving the README a place to live as we want to develop the doc in the new structure

File: docs/pages/lunadefend/legacy/legacy-code.md (hunk 6 above)
Comment (g00b) [reply docs/pages/lunadefend/legacy/legacy-code.md:23]: So I'm thinking of just putting a :::note bubble at the bottom of the page with the text. And then I guess I'll just go ahead and remove the previous page.

**Pull request merged**